Automattic/wp-calypso
# Plans: Recognize new Backup plans in determining Free plan tagline

Resolves `1200412004370260-as-1201311518372696`, `p1HpG7-dpL-p2#comment-50044`.

#### Changes proposed in this Pull Request

* Add **Backup (10 GB)** and **Backup (1 TB)** to the list of Backup products used in determining the tagline for Jetpack Free, so that customers with Backup are no longer asked to add Backup to their site.

#### Testing instructions

1. In Calypso blue, select a Jetpack site with an active subscription for stand-alone **Backup (10 GB)** or **Backup (1 TB)**.
2. Visit the **Upgrades > Plans** page and click the **My Plan** tab.
3. Verify that you see your plan is **Jetpack Free**, and that its description does not mention backups as a reason for upgrading your site (see reference screenshots).

#### Reference screenshots (before / after)

<img width="375" alt="Screen Shot 2021-11-02 at 14 42 23" src="https://user-images.githubusercontent.com/670067/139941729-bf65ab85-34ea-41a5-98f7-cf88ff487043.png"> <img width="375" alt="Screen Shot 2021-11-02 at 14 41 46" src="https://user-images.githubusercontent.com/670067/139941727-0efc0c25-9b1d-4440-901c-1aa8151855e6.png">
